jpegenc/testing/testing.py

```diff
@@ -1,25 +1,28 @@
-
+import py
 import sys
 import os
 import subprocess
 import pytest
 import myhdl
 
-
+sim_is_ok = False
 skip_ref_test = pytest.mark.skipif(reason="skip reference cosimulation")
 if hasattr(sys, '_called_from_test'):
     skip_ref_test = pytest.mark.skipif(
         not pytest.config.getoption("--include-reference"),
         reason="reference tests, needs --include-reference option to run"
     )
 
+skipif_no_sim = pytest.mark.skipif(False, reason="simulator is not available")
+if hasattr(sys, '_called_from_test'):
+    skip_if_no_sim = pytest.mark.skipif(
+        True,
+        reason="simulator is not ok "
+    )
```

Comment:
@cfelton  i added a new marker skip_if_no_sim. I set the _called_from_test attribute to true in the conftest but the test executes and it never skips. I want to put there as a condition the global variable sim_is_ok which is set in the conftest.py if the simulators are available. Do you know how to fix this?